apps/aws-amplify/src/locations/Sidebar.tsx

```diff
@@ -50,9 +52,10 @@ const Sidebar = () => {
         isFullWidth={true}
         isLoading={isLoading}
         isDisabled={isLoading}
-        onClick={buildAmplifyApp}>
-        {isLoading ? 'Building' : 'Build website'}
+        onClick={handleBuildAppActionCall}>
+        {isLoading ? 'Initiating build...' : 'Build now'}
       </Button>
+      <span>Last build started: {lastBuildInitiated}</span>
```

Comment:
It seems odd to me to have this `Button` and `span` rendering within a `Paragraph` component. I would suggest using the `Paragraph` component for the `Last build started` text and then move the button outside.

Reply:
This was left over from another Dev's branch, but agreed elements shouldn't be nested under a paragraph tag